python/mdvtools/dbutils/mdv_server_app.py

```
@@ -1,10 +1,13 @@
 import os
 import time
 import json
+import shutil
+import tempfile
+import zipfile
 import logging
 from sqlalchemy import text
 from sqlalchemy.exc import OperationalError
-from flask import Flask
+from flask import Flask, render_template, jsonify, request
```

Ruff (0.8.2)

`flask.render_template` imported but unused

Remove unused import

(F401)